Makefile

```diff
@@ -308,10 +308,19 @@ clean:
 	rm -rf cairo
 	rm -rf cairo1
 	rm -rf cairo2
+	rm -rf cairo-lang
 
 fuzzer-deps: 
 	cargo +nightly install cargo-fuzz
 
 run-cairo-compiled-fuzzer:
 	cd fuzzer
 	cargo +nightly fuzz run --fuzz-dir . cairo_compiled_programs_fuzzer
+
+CAIRO_LANG_REPO_DIR: cairo-lang
+
+$(CAIRO_LANG_REPO_DIR):
+	git clone --depth=1 https://github.com/starkware-libs/cairo-lang
+
+hint-accountant: $(CAIRO_LANG_REPO_DIR)
```

Comment:
This command is returning me 
```
thread 'main' panicked at 'Failed to read whitelist directory from cairo-lang, did you forget to clone it?
: Os { code: 2, kind: NotFound, message: "No such file or directory" }', hint_accountant/src/main.rs:37:57
```

Also, can we add the`cairo-lang/` folder to the .gitignore?

Reply:
Fixed and added